src/domain/bitcoin/onchain/index.types.d.ts

```diff
@@ -22,14 +25,17 @@ type OnChainTransaction = {
   outs: TxOut[]
 }
 
-type IncomingOnChainTransaction = {
+type BaseOnChainTransaction = {
   confirmations: number
   rawTx: OnChainTransaction
   fee: Satoshis
   createdAt: Date
   uniqueAddresses: () => OnChainAddress[]
 }
 
+type IncomingOnChainTransaction = BaseOnChainTransaction
```

Comment:
This alias will not be type safe as it is not actually changing the signature.

What we want from these types is that you cannot pass an `incomingTx` to a function expecting an `outgoingTx` but I think this doesn't prevent it.